### DISCUSSION TITLE
Freaccmd - Metadata Tags support in command line

### DISCUSSION THREAD

**Discussion by daveinthecity** [enzo1982/freac#306] [Help & Support]
Does **freaccmd** support encoding with ID tags?

e.g. Album, Artist, Copyright, etc. ID tags

**Comment by enzo1982** [comment]
No, freaccmd does not support setting metadata information. It will use the metadata tags from the source files.

Also, at this time, I have no plans to add support for this. This may change at some point, but for now it's best to use 3rd party tools to edit metadata after conversion.
